tests/integration/test_vulnerability_detector/test_scan_results/test_scan_vulnerabilities_triaged_null.py

```diff
@@ -0,0 +1,192 @@
+
+pytestmark = [pytest.mark.server]
+
+# Reference paths
+TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+CONFIGURATIONS_PATH = os.path.join(TEST_DATA_PATH, 'configuration_template')
+TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, 'test_cases')
+TEST_FEEDS_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', 'data', 'feeds')
+TEST_PACKAGES_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', 'data', 'vulnerable_packages')
+
+# Configuration and cases data
+configurations_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_scan_vulnerabilities_triaged_null.yaml')
+t1_cases_path = os.path.join(TEST_CASES_PATH, 'cases_scan_vulnerabilities_triaged_null.yaml')
+
+# Custom paths
+custom_nvd_json_feed_path = os.path.join(TEST_FEEDS_PATH, 'nvd', vd.CUSTOM_NVD_FEED)
+custom_redhat_oval_feed_path = os.path.join(TEST_FEEDS_PATH, 'redhat', vd.CUSTOM_REDHAT_OVAL_FEED)
+custom_redhat_json_feed_path = os.path.join(TEST_FEEDS_PATH, 'redhat', vd.CUSTOM_REDHAT_JSON_FEED)
+
+# Scan NVD test configurations (t1)
+t1_configuration_parameters, t1_configuration_metadata, t1_case_ids = get_test_cases_data(t1_cases_path)
+t1_configurations = load_configuration_template(configurations_path, t1_configuration_parameters,
+                                                t1_configuration_metadata)
+t1_systems = [metadata['system'] for metadata in t1_configuration_metadata]
+
+# Set offline custom feeds configuration
+t1_configurations = update_configuration_template(
+    t1_configurations,  ['CUSTOM_REDHAT_OVAL_FEED', 'CUSTOM_REDHAT_JSON_FEED', 'CUSTOM_NVD_JSON_FEED'],
+    [custom_redhat_oval_feed_path, custom_redhat_json_feed_path, custom_nvd_json_feed_path])
+
+
+@pytest.fixture(scope='function')
+def prepare_scan(metadata, agent_system, mock_agent_with_custom_system):
+    """Prepare the environment to launch the vulnerability scan.
+
+    - Mock an agent with a specified system.
+    - Force full scan.
+
+    Args:
+        metadata (dict): Test case metadata.
+        agent_system (str): System to set to the mocked agent.
+        mock_agent_with_custom_system (fixture): Mock an agent with a custom system.
+    """
+    # Forcing a full-scan
+    agent_db.update_last_full_scan(1, agent_id=mock_agent_with_custom_system)
+
+    yield mock_agent_with_custom_system
+
+
+@pytest.mark.tier(level=1)
+@pytest.mark.parametrize('configuration, metadata, agent_system',
+                         zip(t1_configurations, t1_configuration_metadata, t1_systems), ids=t1_case_ids)
+def test_scan_triaged_null_vulnerabilities(configuration, metadata, agent_system, set_wazuh_configuration_vdt,
+                                           truncate_monitored_files, clean_cve_tables_func, prepare_scan,
+                                           restart_modulesd_function):
+    '''
+    description: Check if a vulnerable package with triaged NULL, is detected with the partial scan.
+
+    test_phases:
```

Comment:
Use the new convention documentation:
- setup
- test
- tierdown

Reply:
Done in [here](https://github.com/wazuh/wazuh-qa/pull/3587/commits/840de2cd5fb93a74e75cb6bbcb6511b3ec15491b)